src/debugger.ts

```diff
@@ -0,0 +1,55 @@
+import * as vscode from "vscode"
+import {CancellationToken, DebugConfiguration, Disposable, ProviderResult, WorkspaceFolder} from "vscode"
```

Comment:
Please format the source code with prettier, that's what me and @gabro currently use even we don't enforce it in CI (but probably should).